tests/ParserInvariantsTest.php

```diff
@@ -76,7 +76,7 @@ public function testNodesAllHaveAtLeastOneChild($filename, Node $sourceFileNode)
      * @dataProvider sourceFileNodeProvider
      */
     public function testEveryNodeSpanIsSumOfChildSpans($filename, Node $sourceFileNode) {
-        $treeElements = iterator_to_array($sourceFileNode->getDescendantNodesAndTokens());
+        $treeElements = iterator_to_array($sourceFileNode->getDescendantNodesAndTokens(), false);
         array_push($treeElements, $sourceFileNode);
```

Comment:
replace stray instances of `array_push` with $array[] - my guess is that it mostly occurs in test code. Just filed an issue tracking here #40